WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/AddBarViewModel.cs

```diff
@@ -1,10 +1,12 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using Xamarin.Forms;
 
 namespace ComparisonEngineUI.ViewModels
 {
     class AddBarViewModel : BaseViewModel
     {
+        //public Command SaveCommand { get; };
```

Comment:
I think later we will. We need to take out the saving logic out of xaml.cs files to viewmodels, that would be the proper way, but don't really know how to do that, but we'll probably use Command instead of Clicked, so it makes sense to leave it for now